packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:io' as io show exit, exitCode;
+
+import 'package:meta/meta.dart';
+
+export 'dart:io'
+    show
+        BytesBuilder,
+        exitCode,
+        GZIP,
+        InternetAddress,
+        IOException,
+        IOSink,
+        HttpClient,
+        HttpClientRequest,
+        HttpClientResponse,
+        HttpHeaders,
+        HttpRequest,
+        HttpServer,
+        HttpStatus,
+        pid,
+        Platform,
+        Process,
+        ProcessException,
+        ProcessResult,
+        ProcessSignal,
+        ProcessStartMode,
+        ServerSocket,
+        stderr,
+        stdin,
+        stdout,
+        Socket,
+        SocketException,
+        SYSTEM_ENCODING,
+        WebSocket,
+        WebSocketTransformer;
```

Comment:
yeah. add a detailed set of comments here explaining the philosophy and purpose of this file. whatever you would tell someone who was about to start working on the tool, refactoring it or whatever other destructive operation might lead them to undo all your hard work. ☺️